patches/minecraft/net/minecraft/client/multiplayer/WorldClient.java.patch

```diff
@@ -17,6 +17,7 @@
          this.field_72988_C = new SaveDataMemoryStorage();
          this.func_72966_v();
          this.func_72947_a();
++        this.func_175643_b();
```

Comment:
I would just call initCapabilities directly (and everywhere else you patch this call in).

Patching in calls to vanilla is always risky. We could be totally wrong about what the method does, and if mojang moves, deletes, or otherwise changes it it screws our patches.